### PR TITLE
Added detailed usage doc for set_openstack_containers role

### PR DESCRIPTION
**Pull request by raukadah**
It includes examples on how it is used in CI.

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: ci_framework/roles/set_openstack_containers/README.md [hunk 1]
```diff
@@ -12,10 +17,72 @@ used during OpenStack services deployment using meta operator.
 * `cifmw_set_openstack_containers_overrides`: Extra container overrides. Defaults to `{}`
 
 ## Examples
+
+Notes: Make sure we installed the operator before updating the environment variable
```
Comment (marios) [inline ci_framework/roles/set_openstack_containers/README.md:21]: `the operator before`

which operator be explicit. do you mean generally all operators?

`Make sure operators are installed before updating the relevant environment variable(s)` 

?

**Review by marios — APPROVED** [review]
lgtm

File: ci_framework/roles/set_openstack_containers/README.md (hunk 1 above)
Comment (raukadah) [reply ci_framework/roles/set_openstack_containers/README.md]: Thank you @marios . I updated the docs. It refers to meta operator.

**Comment by cjeanner** [comment]
/approve

Looks really nice, thanks for the update @raukadah !

https://logserver.rdoproject.org/82/382/09db86d9f7f1cb318f922babc2c207af2844e588/github-check/cifmw-doc/cf760f9/doc_build/html/roles/set_openstack_containers.html

**Pull request merged**